hw/ssi/spi_gpio.c

```diff
@@ -52,14 +119,18 @@ static void spi_gpio_realize(DeviceState *dev, Error **errp)
 
     s->spi = ssi_create_bus(dev, "spi");
 
+    s->mode = 0;
+    s->clk_counter = 0;
+
     s->cs = 1;
     s->clk = 1;
     s->mosi = 1;
+    s->miso = 1;
```

Comment:
```suggestion
    s->cs = true;
    s->clk = true;
    s->mosi = true;
    /* let's assume the first output is 0 */
    s->output_byte = 0; 
```
miso technically is not needed to be a property, as it is the bit of output_byte